foundry.toml

```diff
@@ -50,6 +50,9 @@ dictionary_weight = 40
 include_storage = true
 include_push_bytes = true
 
+[etherscan]
+arbitrum = {key = "${ARBISCAN_TOKEN}"}
```

Comment:
why was this necessary?

Reply:
Maybe it is not, I was trying to fix some issues during deployment